edumfa/lib/token.py

```diff
@@ -2465,32 +2670,40 @@ def get_dynamic_policy_definitions(scope=None):
         this scope.
     :return: The policy definition for the token or only for the scope.
     """
-    from edumfa.lib.policy import SCOPE, MAIN_MENU, GROUP
+    from edumfa.lib.policy import GROUP, MAIN_MENU, SCOPE
 
-    pol = {SCOPE.ADMIN: {},
-           SCOPE.USER: {},
-           SCOPE.AUTH: {},
-           SCOPE.ENROLL: {},
-           SCOPE.WEBUI: {},
-           SCOPE.AUTHZ: {}}
+    pol = {
+        SCOPE.ADMIN: {},
+        SCOPE.USER: {},
+        SCOPE.AUTH: {},
+        SCOPE.ENROLL: {},
+        SCOPE.WEBUI: {},
+        SCOPE.AUTHZ: {},
+    }
     for ttype in get_token_types():
-        pol[SCOPE.ADMIN]["enroll{0!s}".format(ttype.upper())] \
-            = {'type': 'bool',
-               'desc': _("Admin is allowed to initialize {0!s} tokens.").format(ttype.upper()),
-               'mainmenu': [MAIN_MENU.TOKENS],
-               'group': GROUP.ENROLLMENT}
-
-        conf = get_tokenclass_info(ttype, section='user')
-        if 'enroll' in conf:
-            pol[SCOPE.USER]["enroll{0!s}".format(ttype.upper())] = {
-                'type': 'bool',
-                'desc': _("The user is allowed to enroll a {0!s} token.").format(ttype.upper()),
-                'mainmenu': [MAIN_MENU.TOKENS],
-                'group': GROUP.ENROLLMENT}
+        pol[SCOPE.ADMIN][f"enroll{ttype.upper()!s}"] = {
+            "type": "bool",
+            "desc": _("Admin is allowed to initialize {0!s} tokens.").format(
```

Comment:
Can we fix some expression like this one? could imagine that we have to encapsulate the f-string in `()` and then apply the `_` translation?
Think we also have to check the translations...

Reply:
When it comes to localization, we need to keep using `.format`, otherwise it won't work.

Reply:
Ok. Thanks.